extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -14,7 +14,7 @@ export interface DbServerGraphData extends GraphData {
 	};
 }
 
-export const serversQuery = `where type == "${azureResource.AzureResourceType.sqlServer}"`;
+export const serversQuery = `where type == "${azureResource.AzureResourceType.sqlServer}" and kind != "v12.0,analytics"`;
```

Comment:
So now we won't show Synapse SQL Pool servers at all, which probably isn't what we want. We should still show them - just be fixing it so that when it is a SQL Pool server we use the SQL endpoint instead of the FQDN. (this was the fix suggestion I had before with changing the fullName property in convertResource)

Reply:
This will require making a second query call to grab the object that contains the connectivityEndpoints itself (you have to use the Microsoft.synapse query), then replacing the entry with the correct one (assuming we want to keep the first entry of the Synapse pool from the server query)

Reply:
Does properties.connectivityEndpoints not get returned as part of the properties blob from the original query?

If that's the case then you'll probably need to modify the query to do the join like I suggested elsewhere (which is probably safer to do anyways so not a bad idea regardless).

Reply:
It does not at all, so a join would have to be done.

Reply:
@Charles-Gagnon Looks like the resources call does not currently support the union operator at all (I used "| union (resources | where type == "etc")" at the end of the query which led to an invalid query error), so two separate queries will have to be done, but their results can be combined.

Reply:
Hm, that's unfortunate, but two separate queries is fine for if you can get them working, although I feel like there's probably a way that we're just missing so might be worth digging into a bit more at some point (the docs on this stuff are horrible, almost nothing out there that I can find)